endpoints/openrtb2/amp_auction.go

```diff
@@ -240,6 +254,37 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 		return
 	}
 
+	if isRejectErr {
+		labels, ao = rejectAmpRequest(*rejectErr, w, reqWrapper, labels, ao, errL)
+		return
+	}
+
+	labels, ao = sendAmpResponse(w, response, reqWrapper, labels, ao, errL)
+}
+
+func rejectAmpRequest(
+	rejectErr hookexecution.RejectError,
+	w http.ResponseWriter,
+	reqWrapper *openrtb_ext.RequestWrapper,
+	labels metrics.Labels,
+	ao analytics.AmpObject,
+	errs []error,
+) (metrics.Labels, analytics.AmpObject) {
+	response := &openrtb2.BidResponse{NBR: openrtb3.NoBidReason(rejectErr.NBR).Ptr()}
+	ao.AuctionResponse = response
+	ao.Errors = append(ao.Errors, rejectErr)
+
+	return sendAmpResponse(w, response, reqWrapper, labels, ao, errs)
+}
+
+func sendAmpResponse(
```

Comment:
Nitpick: Could you re-format this function so the parameters are all on one line to follow the patterns of other functions in this file?